src/types/snackbar.ts

```diff
@@ -0,0 +1,18 @@
+export interface SnackbarPayload {
+  color: string;
+  message: string;
+}
+
+export interface SnackbarState extends SnackbarPayload {
+  isOpen: boolean;
+}
+
+export enum SnackbarActions {
+  push = "pushNewNotification",
+  reset = "resetOpenState"
+}
+
+export enum SnackbarMutations {
+  set = "setSnackbarState",
+  reset = "resetSnackbarState"
+}
```

Comment:
Please change the key to match its value.

Reply:
not my work